core/src/smartcontracts/isi/asset.rs

```diff
@@ -319,12 +376,26 @@ pub mod isi {
             Mintable::Infinitely => Ok(()),
             Mintable::Not => Err(Error::Mintability(MintabilityError::MintUnmintable)),
             Mintable::Once => {
```

Comment:
if we'll allow for initial value to be set in the asset registration request (as far as I see this is the case) then I don't see a need for `Mintable` to be enum, rather it can be a `bool` since there is no need for `Mintable::Once` any longer